include/gatekeeper_gk.h

```diff
@@ -19,12 +19,17 @@
 #ifndef _GATEKEEPER_GK_H_
 #define _GATEKEEPER_GK_H_
 
+#include <rte_atomic.h>
+
 /* Configuration for the GK functional block. */
 struct gk_config {
+	rte_atomic32_t	ref_cnt;
```

Comment:
Move `ref_cnt` to the end of the struct and apply the same comment used in `struct net_config`.